### ISSUE TITLE
Screen lock issues in `v0.40`

### ISSUE THREAD

**Issue by Zeioth** [hyprwm/Hyprland#6034]
### Hyprland Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch  at commit cba1ade848feac44b2eda677503900639581c3f4  (props: bump version to 0.40.0).
Date: Sat May 4 15:42:32 2024
Tag: v0.40.0, commits: 4606

flags: (if any)


System Information:
System name: Linux
Node name: zeioht-pc
Release: 6.9.0-rc7-273-tkg-eevdf
Version: #1 SMP PREEMPT_DYNAMIC TKG Wed, 08 May 2024 21:44:41 +0000


GPU information: 
04:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] [1002:67df] (rev ef) (prog-if 00 [VGA controller])
2d:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Navi 22 [Radeon RX 6700/6700 XT/6750 XT / 6800M/6850M XT] [1002:73df] (rev c1) (prog-if 00 [VGA controller])


os-release: NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://gitlab.archlinux.org/groups/archlinux/-/issues"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo


plugins:
```

</details>

### Bug or Regression?

Regression

### Description

Multiple bugs related to screen lock have been introduced in v0.40:

## Inconsistent behavior
* Not all screens are correctly locked: Out of my 6 monitors, only 5 of them are actually locked. The 6º of them still display it's content.

## The screen lock in new unwated scenarios
* While playing videogames (When using the gamepad instead of the mouse).
* While playing video (all screens lock except the one displaying the video, which lock the controls).
* While playing audio (v0.40 seem to break `sway-audio-idle-inhibit`).

## Inconsistent behavior
It reproduces 100% of the time in my machine just by locking it.

### Scenario 1
* Play a videogame using a xbox gamepad

### Scenario 2
* Play a video while having multiple monitors attached.

### Scenario 3
* Play music while running `sway-audio-idle-inhibit`. The screen will lock anyway.

**Comment by Zeioth** [comment]
Thank you, I will be testing the next weeks and close the issue when all is good 👍

**Comment by sofijacom** [comment]
> While playing audio (v0.40 seem to break sway-audio-idle-inhibit).

Yes, my screen also locks when playing sound. `sway-audio-idle` installed from Aur `sway-audio-idle-inhibit-git`

**Comment by sofijacom** [comment]
> > При воспроизведении звука (версия 0.40, похоже, отключает sway-audio-idle-inhibit).
> 
> Да, мой экран также блокируется при воспроизведении звука. `sway-audio-idle` установлен с Aur `sway-audio-idle-inhibit-git`

I fixed this by increasing the time in `hypridle.conf` 120 seconds by 300 seconds
`# Screen_brightness
listener {
	timeout = 300
	on-timeout = brightnessctl s 20%
	on-resume = brightnessctl s 50%
}

# Screen_lock
listener {
    timeout = 600
    on-timeout = hyprlock
    on-resume = notify-send "Welcome back to Hyprland!"
}
`

**Comment by sofijacom** [comment]
> > При воспроизведении звука (версия 0.40, похоже, отключает sway-audio-idle-inhibit).
> 
> Да, мой экран также блокируется при воспроизведении звука. `sway-audio-idle` установлен с Aur `sway-audio-idle-inhibit-git`
> 
> Я исправил это, увеличив время со 120 секунд до 300 секунд в `hypridle.conf`

**Comment by Zeioth** [comment]
I'm gonna keep testing a few more days but everything seems good in `hyprland-git`.

Everything seems back to normal.
